tests/integration/components/registries/registries-services-list/component-test.ts

```diff
@@ -0,0 +1,332 @@
+                'The contact link text is correct.');
+    });
+
+
+    test('the registries services list DARPA ASIST', async function(this: TestContext, assert) {
+        // Given the component is rendered
+        await render(hbs`<RegistriesServicesList />`);
+
+        // Given I find the node
+        const node = document.querySelector('[data-test-registries-list-row-one] > div:nth-child(1) > a ');
+
+        // Then I validate the link
+        assert.dom(node)
+            .hasAttribute('href',
+                'https://osf.io/registries/darpaasist',
+                'The a href link is correct.');
+
+        // And I validate the link aria-label
+        assert.dom(node)
+            .hasAttribute('aria-label',
+                'DARPA ASIST Registry',
+                'The a aria-label is correct.');
+
+        // And I validate the data analytics name
+        assert.dom(node)
+            .hasAttribute('data-analytics-name',
+                'asist - Registry',
+                'The a aria-label is correct.');
+
+        // Given I find the image node
+        const imageNode = node?.querySelector('img');
+
+        // Then I validate the image source
+        assert.dom(imageNode)
+            .hasAttribute('src',
+                '/engines-dist/registries/assets/img/provider_logos/ASIST_logo.png',
+                'The image src is correct.');
+
+        // And I validate the image alt tag
+        assert.dom(imageNode)
+            .hasAttribute('alt',
+                'DARPA ASIST Registrylogo',
+                'The image alt tag is correct.');
+    });
+
+    test('the registries services list Character Lab Registry', async function(this: TestContext, assert) {
+        // Given the component is rendered
+        await render(hbs`<RegistriesServicesList />`);
+
+        // Given I find the node
+        const node = document.querySelector('[data-test-registries-list-row-one] > div:nth-child(2) > a ');
+
+        // Then I validate the link
+        assert.dom(node)
+            .hasAttribute('href',
+                'https://osf.io/registries/characterlabregistry',
+                'The a href link is correct.');
+
+        // And I validate the link aria-label
+        assert.dom(node)
+            .hasAttribute('aria-label',
+                'Character Lab Registry',
+                'The a aria-label is correct.');
+
+        // And I validate the data analytics name
+        assert.dom(node)
+            .hasAttribute('data-analytics-name',
+                'characterlab - Registry',
+                'The a aria-label is correct.');
+
+        // Given I find the image node
+        const imageNode = node?.querySelector('img');
+
+        // Then I validate the image source
+        assert.dom(imageNode)
+            .hasAttribute('src',
+                // eslint-disable-next-line max-len
+                '/engines-dist/registries/assets/img/provider_logos/CharacterLab_logo-1794e99e73d6b8208ba3a8ba314a2e27.png',
```

Comment:
I'm not sure if the `1794e99e73d6b8208ba3a8ba314a2e27` fingerprint ID is needed in this src reference? It seems odd that `ASIST_logo` before this didn't need it, but these ones after need it?